mods/d2k/maps/atreides-02a/atreides02a.lua

```diff
@@ -0,0 +1,124 @@
+
+HarkonnenBase = { HConyard, HPower1, HPower2, HBarracks }
+
+HarkonnenReinforcements = { }
+HarkonnenReinforcements["Easy"] =
+{
+	{ "light_inf", "trike" },
+	{ "light_inf", "trike" },
+	{ "light_inf", "light_inf", "light_inf", "trike", "trike" }
+}
+
+HarkonnenReinforcements["Normal"] =
+{
+	{ "light_inf", "trike" },
+	{ "light_inf", "trike" },
+	{ "light_inf", "light_inf", "light_inf", "trike", "trike" },
+	{ "light_inf", "light_inf" },
+	{ "light_inf", "light_inf", "light_inf" },
+	{ "light_inf", "trike" },
+}
+
+HarkonnenReinforcements["Hard"] =
+{
+	{ "trike", "trike" },
+	{ "light_inf", "trike" },
+	{ "light_inf", "trike" },
+	{ "light_inf", "light_inf", "light_inf", "trike", "trike" },
+	{ "light_inf", "light_inf" },
+	{ "trike", "trike" },
+	{ "light_inf", "light_inf", "light_inf" },
+	{ "light_inf", "trike" },
+	{ "trike", "trike" }
+}
+
+HarkonnenAttackPaths =
+{
+	{ HarkonnenEntry1.Location, HarkonnenRally1.Location },
+	{ HarkonnenEntry1.Location, HarkonnenRally3.Location },
+	{ HarkonnenEntry2.Location, HarkonnenRally2.Location },
+	{ HarkonnenEntry2.Location, HarkonnenRally4.Location }
+}
+
+HarkonnenAttackDelay = { }
+HarkonnenAttackDelay["Easy"] = DateTime.Minutes(5)
+HarkonnenAttackDelay["Normal"] = DateTime.Minutes(2) + DateTime.Seconds(40)
+HarkonnenAttackDelay["Hard"] = DateTime.Minutes(1) + DateTime.Seconds(20)
+
+HarkonnenAttackWaves = { }
+HarkonnenAttackWaves["Easy"] = 3
+HarkonnenAttackWaves["Normal"] = 6
+HarkonnenAttackWaves["Hard"] = 9
```

Comment:
Just a note for the future: you can write it like this as well, which looks a bit nicer IMO:

```lua
HarkonnenAttackWaves =
{
	Easy = 3,
	Normal = 6,
	Hard = 9,
}
```

Other examples are in gdi05a.